yaml/boundaries.yaml

```diff
@@ -189,6 +213,22 @@ filters:
       <<: *output_properties
```

Comment:
For the `any: place` filters above here, add inline documentation that explains the LUA imputes the place values from the relation metadata on load into Postgres.